docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/ruby-on-rails.md

```diff
@@ -98,6 +98,8 @@ The final step is to configure the exporter host, service and application name.
 
 * `OTEL_RESOURCE_ATTRIBUTES=application=APPLICATION_NAME` - configure the application name. This will appear as a tracing application name in Sumo Logic. Additional attributes can be added here as comma separated key=value pairs.
 
+* {@import ../../../../reuse/otel-deployment-environment-tag.md}
```

Comment:
Please put it as a part of `OTEL_RESOURCE_ATTRIBUTES`. It's not separated environment variable.